test/module/irohad/model/converters/json_query_factory_test.cpp

```diff
@@ -181,4 +185,24 @@ TEST(QuerySerializerTest, get_role_permissions){
   auto val = queryGenerator.generateGetRolePermissions();
   val->signature = generateSignature(42);
   runQueryTest(val);
-}
\ No newline at end of file
+}
+
```

Comment:
I feel we need to add conversion tests for `Pager` with Json and Proto.

Reply:
Fixed it